.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+## Description
+
+✅ ...
+✅ ...
+
+## References
+
+[Trello-XXXX](https://bcg-platinion.atlassian.net/browse/ER-XXXX)
+
+## Types of changes
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Refactor (non-breaking change which improves existing functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+<!--
+  ⚠️ IMPORTANT ⚠️
+  - DELETE EVERYTHING FROM THIS COMMENT ONWARDS BEFORE MERGING PR
+  - Squash merge with the following message:
+    <type>(<epicCode>): [<ticketCode>] (scope) - shortText
+    i.e.: feat(er-324): [er-2563] (AvtaleGiro) - Agreement summary screen
+-->
+
+## Visuals
+
+![Alt Text](https://media.giphy.com/media/3oKIPnAiaMCws8nOsE/giphy.gif)
+
+## Checklist:
+
+<!--- Ensure quality. Put an `x` in all that apply -->
+
+- [ ] I have added unit tests to cover my changes
+- [ ] I have tested my code locally for app - ios
+- [ ] I have tested my code locally for app - android
+- [ ] I have tested my code locally for web
+- [ ] I have updated the documentation (created stories where apply) accordingly
+- [ ] I updated the ticket status
+- [ ] I confirm that new and existing tests passed
```

Comment:
Could perhaps have one check for "I have updated necessary library patch versions" for instance, either in checklist or as as default check option in TODOs

Reply:
added another check under checklist